docs/layers/scenegraph-layer.md

```diff
@@ -71,6 +71,10 @@ Can be a URL of an object. You need to provide the `fetch` function to load the
 Can also be a luma.gl [ScenegraphNode](http://uber.github.io/luma.gl/#/documentation/api-reference/), or a `Promise` that resolves to one.
 The layer calls _delete()_ on _scenegraph_ when a new one is provided or the layer is finalized.
 
+##### `scenegraphLoader` (Object, optional)
+
+If you provide a URL for `scenegraph` you can optionally provide a `scenegraphLoader` to load the URL.
+If not provided `loaders.gl` will select one automatically.
```

Comment:
Explain what the "automatic" behavior is? Why do we need this feature to support additional formats instead of just calling `registerLoaders`?

Reply:
Done